app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -308,7 +318,11 @@
                         ga4_link: {
                           "event_name": "navigation",
                           "type": "header menu bar",
-                          "index": "2.2.#{index + 1}",
+                          "index": {
+                            "index_section": 2,
```

Comment:
This is currently the same section number as `Government Activity` and I think it needs to increment

Reply:
Great spot, thanks, have updated.